invenio_app_rdm/config.py

```diff
@@ -746,3 +747,11 @@ def files_rest_permission_factory(obj, action):
     "tif": "tiff",
     "tiff": "tiff",
 }
+
+ROBOTS_TXT = """\
```

Comment:
was adding a new config variable consulted with anyone ? did we consider any other implementation options?

Reply:
Other potential solutions that came to my mind are:
- Serve a robots.txt file (instead of answering the request with text).
- Define the robots .txt in the deploy configuration (I don't know properly how the deploy configuration works right now in Zenodo, or I have access).
This solution was considered the best after talking with @alexdutton.